### PR TITLE
Add support for websockets 11

### PR DESCRIPTION
**Pull request by alanhdu**
Upstream has officially added support for websockets 11, so let's add it here as well.

https://github.com/graphql-python/gql/pull/424

### CODE REVIEW

**Comment by alanhdu** [comment]
@conda-forge-admin, please rerender

**Comment by bollwyvl** [comment]
Seems like a lot of fail related to websockets...

File: recipe/meta.yaml [hunk 1]
```diff
@@ -49,7 +49,7 @@ outputs:
         - requests-toolbelt >=0.9.1,<1
         - urllib3 >=1.26,<2
         # websockets
-        - websockets >=10,<11
+        - websockets >=10,<12
```
Comment (bollwyvl) [inline recipe/meta.yaml:52]: It looks like while upstream's `master` has relaxed this pin, `3.4.1` still has the upper bound. 

https://github.com/graphql-python/gql/blob/v3.4.1/setup.py#L50-L53

**Comment by alanhdu** [comment]
Yeah, it look slike I jumped the gun here. I'll close this and re-open if needed after the next gql is released.

**Pull request closed without merging**